cmd/git/git.go

```diff
@@ -1142,6 +1142,26 @@ func (j *DSGit) SetParentCommitFlag(richItem map[string]interface{}) (err error)
 	return
 }
 
+func (j *DSGit) dedupAuthors(inContributors []insights.Contributor) (outContributors []insights.Contributor) {
```

Comment:
AFAIK - named return values should be avoided in go?

Reply:
Also heard this, but I actually like them and don't see anything wrong in this particular case. can change if you want this?

Reply:
It's okay, it is applicable for longer method/functions as it can make things unreadable in that case.

Reply:
yes, but sometimes I actually prefer to have a named variable and then implicit `return`, in case of any change to logic I don't have to change all `returns` in that case, and I (personally) don't think this makes code less readable, I think the opposite - you see from the function signature what the return parameter names are, so when you reach any return you know what is returned and don't need to add complex return line in all cases. Anyway, as you agreed, will keep it that way in this case.

Reply:
yes